pkgs/os-specific/linux/tuxedo-keyboard/default.nix

```diff
@@ -21,7 +22,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Full color keyboard driver for tuxedo computers laptops";
     homepage = "https://github.com/tuxedocomputers/tuxedo-keyboard/";
-    license = licenses.gpl2;
+    license = licenses.gpl3;
```

Comment:
Is that gpl3Only or gpl3Plus?

Reply:
gpl3Plus :) Thanks for pointing it out. Completely forgot about that.